services/event_service.go

```diff
@@ -83,6 +85,11 @@ func (e *EventService) CreateAppEvent(ctx context.Context, newMessage *models.Ev
 		return nil, util.NewServiceError(http.StatusBadRequest, errors.New("app has no configured endpoints"))
 	}
 
+	// TODO(daniel): consider adding this check
+	// if app.GroupID != g.UID {
+	//	return nil, util.NewServiceError(http.StatusUnauthorized, errors.New("unauthorized to access app"))
+	// }
```

Comment:
Why was this commented out?

Reply:
It's a TODO, maybe we should consider adding it cc @Subomi.
Because `app_id` is specified in the event request body we should verify that the app belongs to the group that the request was authenticated for.